ceph-releases/ALL/rhel8/daemon/content_sets.yml

```diff
@@ -0,0 +1,25 @@
+# This is a file defining which content sets (yum repositories) are needed to
+# update content in this image. Data provided here helps determine which images
+# are vulnerable to specific CVEs. Generally you should only need to update this
+# file when:
+#    1. You start depending on a new product
+#    2. You are preparing new product release and your content sets will change
+#
+# See https://mojo.redhat.com/docs/DOC-1023066 for more information on
+# maintaining this file and the format and examples
+#
+# You should have one top level item for each architecture being built. Most
+# likely this will be x86_64 and ppc64le initially.
+---
+x86_64:
+- rhel-8-server-rpms
+- rhel-8-server-extras-rpms
+- rhel-8-server-rhceph-4-tools-rpms
+- rhel-8-server-rhceph-4-mon-rpms
+- rhel-8-server-rhceph-4-osd-rpms
+ppc64le:
+- rhel-8-for-power-le-rpms
+- rhel-8-for-power-le-extras-rpms
+- rhel-8-server-rhceph-4-for-power-le-tools-rpms
+- rhel-8-server-rhceph-4-for-power-le-mon-rpms
+- rhel-8-server-rhceph-4-for-power-le-osd-rpms
```

Comment:
From what I can tell of RCM's RHEL 8 Pulp plans, here is what we'll need in this file:

```yaml
---
x86_64:
- rhel-8-x86_64-rpms
- rhel-8-x86_64-extras-rpms
- rhceph-4-tools-for-rhel-8-x86_64-rpms
- rhceph-4-mon-for-rhel-8-x86_64-rpms
- rhceph-4-osd-for-rhel-8-x86_64-rpms
ppc64le:
- rhel-8-ppc64le-rpms
- rhel-8-ppc64le-extras-rpms
- rhceph-4-tools-for-rhel-8-ppc64le-rpms
- rhceph-4-mon-for-rhel-8-ppc64le-rpms
- rhceph-4-osd-for-rhel-8-ppc64le-rpms
```

if it turns out we need more changes here, we can include them later.